frontend/src/Pages/CommentCard/CommentGrid.js

```diff
@@ -0,0 +1,62 @@
+import React, {Component} from 'react';
+import {
+  Container,
+  Row,
+  Col
+} from 'react-bootstrap';
+/*import {Card} from 'react-bootstrap';*/
+import './CommentCard.css'
+import CommentCard from './CommentCard';
+
+
+class CommentGrid extends Component{
```

Comment:
It will be good if you could use the - function style to create react component. For e.g. 
function CommentGrid () {
  const people = [...]
}
instead of
class CommentGrid () {
  ///
}